### PR TITLE
Removes the suppose-to-be unused shuttle wall path

### PR DESCRIPTION
**Pull request by ChangelingRain**
snip snip

### CODE REVIEW

**Comment by KorPhaeron** [comment]
@ChangelingRain Travis

**Comment by ChangelingRain** [comment]
@ShizCalev please save me from all these maps using an outdated path

**Comment by lzimann** [comment]
`loading RandomZLevels/moonoutpost19.dmm`
error on this map

File: code/game/gamemodes/clock_cult/clock_helpers/fabrication_helpers.dm [hunk 1]
```diff
@@ -27,9 +27,6 @@
 /turf/closed/wall/mineral/cult/fabrication_vals(mob/living/user, obj/item/clockwork/replica_fabricator/fabricator, silent) //no metal
 	return list("operation_time" = 80, "new_obj_type" = /turf/closed/wall/clockwork, "power_cost" = POWER_WALL_TOTAL, "spawn_dir" = SOUTH)
 
-/turf/closed/wall/shuttle/fabrication_vals(mob/living/user, obj/item/clockwork/replica_fabricator/fabricator, silent) //two sheets of metal
```
Comment (ShizCalev) [inline code/game/gamemodes/clock_cult/clock_helpers/fabrication_helpers.dm]: Shouldn't this be changed to /turf/closed/wall/mineral/titanium instead then?

Comment (ChangelingRain) [reply code/game/gamemodes/clock_cult/clock_helpers/fabrication_helpers.dm]: it's covered by an existing one; `/turf/closed/wall/mineral/fabrication_vals(mob/living/user, obj/item/clockwork/replica_fabricator/fabricator, silent)`

Comment (ShizCalev) [reply code/game/gamemodes/clock_cult/clock_helpers/fabrication_helpers.dm]: Oh lovely, was looking for /titanium and missed it.

**Comment by Bawhoppen** [comment]
I HAVE KILLED THIS SO MANY TIMES WHY DOES IT KEEP COMING BACK

**Review by Cyberboss — APPROVED** [review]
By the father, the spirit, and the holy titanium

**Pull request merged**